lib/index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     'jquery-ember-run': require('./rules/jquery-ember-run'),
     'local-modules': require('./rules/local-modules'),
     'named-functions-in-promises': require('./rules/named-functions-in-promises'),
+    'new-ember-data-packages': require('./rules/new-ember-data-packages'),
```

Comment:
Sorry about this one, but I **really really really** dislike using "new" in the name of things (yes I know the `new-module-imports` has existed for a long time, but we can't really fix that since its already been released). Can we think up a different rule name here? Just brainstorming:

* require-rfc-395-ember-data-imports
* no-ember-data-global-imports
* no-ds-global

Have any better ideas?

Reply:
Indeed, I picked that name in reference to the `new-modules-imports` rule.

> I really really really dislike using "new" in the name of things 

😂 I can understand that. So called `new` things tend to be outdated pretty fast. Especially in the javascript / front-end world.

It could also be: `no-old-ember-data-imports`.

But `no-ember-data-global-imports` sounds better. I'll go for that!

Reply:
👍 	sounds good!

Reply:
It is renamed by now!